src/components/button/Button.tsx

```diff
@@ -14,9 +14,14 @@ export type ButtonProps = {
   /** If true, substitutes the button content by an animated loader */
   loading?: boolean;
   type?: 'button' | 'reset' | 'submit';
-   /** The variant to use*/
-  variant?: 'primary' | 'secondary' | 'tertiary' | 'destructive';
   onClick?: (event: React.MouseEvent<HTMLElement>) => void;
+   /** The variant to use*/
+  variant?: 'primary' | 'secondary' | 'tertiary';
```

Comment:
we should keep "destructive" and have a fallback to work, and be backward compatible